### ISSUE TITLE
face_distance function

### ISSUE THREAD

**Issue by MichaelSchroter** [ageitgey/face_recognition#1093]
* face_recognition version: 1.2.3
* Python version: 3.5.3
* Operating System: Debian Linux 9.9

Hi,

Just wanted to know the `face_distance` function. So if the distance is low does that mean they are more similar?

Thanks & Best Regards

Michael

**Comment by ageitgey** [comment]
Yes, that is correct. 0 distance means they are probably the same image. 

Here's a bit more information about the range: https://github.com/ageitgey/face_recognition/wiki/Calculating-Accuracy-as-a-Percentage
